aviary/variable_info/variable_meta_data.py

```diff
@@ -2414,14 +2426,15 @@
 add_meta_data(
     Aircraft.Engine.Gearbox.SHAFT_POWER_DESIGN,
     meta_data=_MetaData,
-    historical_name={"GASP": 'INPROP.HPMSLS',  # max sea level static horsepower, hp
-                     "FLOPS": None,
-                     "LEAPS1": None,
-                     },
+    historical_name={
+        "GASP": 'INPROP.HPMSLS',  # max sea level static horsepower, hp
```

Comment:
the gasp variable doesn't have the same units as the aviary variable I think this will cause problems in the fortran to aviary converter